pydevices/acq400Devices/acq400_base.py

```diff
@@ -23,26 +23,28 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
-import sys
 import threading
 import socket
 import time
-import inspect
-
-import numpy as np
+from queue import Queue, Empty
```

Comment:
pythton2 incompatible.. and import full modules
https://www.mdsplus.org/index.php?title=pydevice_dev

Reply:
This is now done.